### ISSUE TITLE
Documentation to be added for how to run unit tests

### ISSUE THREAD

**Issue by daidoji** [hyperledger-labs/did-webs-resolver#51]
When I try to run the unit tests naively with `pytest` alone I get the following error.

```(keri) daidoji@LAPTOP-5KEUASIG:~/ssi/did_webs/did-webs-resolver$ pytest
=================================================================================================================== test session starts ====================================================================================================================platform linux -- Python 3.12.2, pytest-8.0.2, pluggy-1.4.0
rootdir: /home/daidoji/ssi/did_webs/did-webs-resolver
collected 1 item / 2 errors

========================================================================================================================== ERRORS ==========================================================================================================================_____________________________________________________________________________________________________ ERROR collecting tests/dkr/core/test_didding.py ______________________________________________________________________________________________________ImportError while importing test module '/home/daidoji/ssi/did_webs/did-webs-resolver/tests/dkr/core/test_didding.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../.pyenv/versions/3.12.2/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/home/daidoji/did_webs/did-webs-resolver/tests/dkr/core/test_didding.py:27: in <module>
    ???
E   ModuleNotFoundError: No module named 'common'
_____________________________________________________________________________________________________ ERROR collecting tests/dkr/core/test_webbing.py ______________________________________________________________________________________________________ImportError while importing test module '/home/daidoji/ssi/did_webs/did-webs-resolver/tests/dkr/core/test_webbing.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../.pyenv/versions/3.12.2/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/home/daidoji/did_webs/did-webs-resolver/tests/dkr/core/test_webbing.py:5: in <module>
    ???
E   ModuleNotFoundError: No module named 'common'
===================================================================================================================== warnings summary =====================================================================================================================../../../.pyenv/versions/3.12.2/envs/keri/lib/python3.12/site-packages/falcon/media/__init__.py:4
  /home/daidoji/.pyenv/versions/3.12.2/envs/keri/lib/python3.12/site-packages/falcon/media/__init__.py:4: DeprecationWarning: 'cgi' is deprecated and slated for removal in Python 3.13
    from .handlers import Handlers

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================================================================================= short test summary info ==================================================================================================================ERROR tests/dkr/core/test_didding.py
ERROR tests/dkr/core/test_webbing.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!=============================================================================================================== 1 warning, 2 errors in 0.62s ===============================================================================================================(keri) daidoji@LAPTOP-5KEUASIG:~/ssi/did_webs/did-webs-resolver$ python -V
Python 3.12.2```

I'm running on python 3.12 with a fresh install to a virtualenv.  Also checked for python 3.11 and 3.10 but same error.  How should tests be run?

**Comment by 2byrds** [comment]
Thank you for reporting @daidoji I will try to reproduce soon and will respond here.

**Comment by 2byrds** [comment]
fixed by #52 . sorry for the inconvenience @daidoji , the common file was missing :\
